src/server/v2.0/handler/project.go

```diff
@@ -547,6 +559,11 @@ func (a *projectAPI) UpdateProject(ctx context.Context, params operation.UpdateP
 		}
 	}
 
+	// ignore metadata.proxy_speed_kb for non-proxy-cache project
+	if params.Project.Metadata != nil && !p.IsProxy() {
+		params.Project.Metadata.ProxySpeedKb = nil
+	}
+
```

Comment:
Here need to ignore the proxy_speed_kb setting for non proxy cache project. But the code logic here is ignoring the proxy cache project.

Reply:
nice catch! updated.